app/assets/javascripts/dashboard/callout.js

```diff
@@ -52,13 +54,16 @@ DashboardCallout = function () {
   };
 
   pumiCheckbox = function (type, object) {
-    return '<div class="form-check">' +
+    var inputTag = '<div class="form-check">' +
     '<input class="form-check-input ' + type + '-ids" type="checkbox" ' +
     'value=' + object.id + ' name="callout[' + type + '_ids][]" ' +
-    'id="callout_' + type + '_' + object.id + '">' +
-    '<label class="string optional form-check-label"' +
+    'id="callout_' + type + '_' + object.id + '">';
+
+    var labelTag = '<label class="string optional form-check-label"' +
     'for="callout_' + type + '_' + object.id + '">'
-    + object.name_km + '&nbsp;(' + object.name_en + ')</label></div>';
+    + object['name_' + _this.locale] + '</label></div>';
```

Comment:
handle the case where locale is not one of `en` or `km`. In this case it should always return the english